sdk/src/types.rs

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    client::AccountAddress,
+    crypto::{
+        ed25519::{Ed25519PrivateKey, Ed25519PublicKey},
+        traits::Uniform,
+    },
+    transaction_builder::TransactionBuilder,
+    types::transaction::{authenticator::AuthenticationKey, RawTransaction, SignedTransaction},
+};
+
+pub use diem_types::*;
+
+pub struct LocalAccount {
```

Comment:
This is most likely only be used in testing code, maybe create a package named testing and put all testing related stuff in it.
I plan to move python sdk's LocalAccount into testing module. But because I started LocalAccount at diem package, can't really remove it from diem package now.

Reply:
Thankfully due to how we're versioning the rust sdk we don't have to worry about being stuck with things forever (at least not yet). Generally I'm finding that having too many packages makes things a bit difficult to manage so I'm hesitant to make another package just for this atm. We could look into shifting it into another module though.

And while I agree that its probably only going to be used for testing for us, this should provide a good tool to help folks prototype stuff